File: example/src/main/java/com/github/vivchar/example/widgets/MyAdapter.java
package com.github.vivchar.example.widgets;

import com.github.vivchar.rendererrecyclerviewadapter.RendererRecyclerViewAdapter;

public class MyAdapter extends RendererRecyclerViewAdapter {
    public MyAdapter() {
        super();
        registerViewFinder(CustomViewFinder::new);
    }
}

File: example/src/main/java/com/github/vivchar/example/widgets/MyAdapter.kt
package com.github.vivchar.example.widgets

import com.github.vivchar.rendererrecyclerviewadapter.RendererRecyclerViewAdapter

open class MyAdapter : RendererRecyclerViewAdapter() {

	init {
		/* an issue after kotlin converting: https://stackoverflow.com/a/50222496/4894238 */
		registerViewFinder { CustomViewFinder(it) }
	}
}